make.js

```diff
@@ -1067,7 +899,6 @@ target.b2g = function() {
   echo();
   echo('### Building B2G (Firefox OS App)');
   var B2G_BUILD_CONTENT_DIR = B2G_BUILD_DIR + '/content/';
-  target.bundle();
```

Comment:
This has been removed, but nothing has been put back for it. Shouldn't we call a generic bundle target here since there is no preprocessing for the B2G target?